pkg/vpcmodel/grouping.go

```diff
@@ -643,22 +643,28 @@ func listEndpointElemStrWithConfig(eps []EndpointElem, fn func(ep EndpointElem,
 }
 
 func (g *groupedExternalNodes) String() string {
+	// 1 gets list and a union of all IPBlocks
+	unionBlock := g.toIPBlocks()
+	// 2. print a list s.t. each element contains either a single cidr or an ip range
+	return strings.Join(unionBlock.ListToPrint(), commaSeparator)
+}
+
+func (g *groupedExternalNodes) toIPBlocks() *ipblock.IPBlock {
```

Comment:
```suggestion
func (g *groupedExternalNodes) toIPBlock() *ipblock.IPBlock {
```
To be more consistent

Reply:
done